rllib/core/learner/learner.py

```diff
@@ -214,8 +214,8 @@ class Learner(Checkpointable):
 
             class MyLearner(TorchLearner):
 
-               def compute_loss(self, fwd_out, batch):
-                   # compute the loss based on batch and output of the forward pass
+               def compute_losses(self, fwd_out, batch):
+                   # Compute the loss based on batch and output of the forward pass
                    # to access the learner hyper-parameters use `self._hps`
```

Comment:
Remove `_hps`. Hyperparameters are not used anymore.

Reply:
Great catch! Will fix.

Reply:
fixed